resources/views/layouts/main.blade.php

```diff
@@ -9,9 +9,8 @@
 @include('includes.spinner')
 @include('includes.navbar')
 
-
-
 @yield('content')
+@yield('detail')
```

Comment:
why did you add it to the template  I didn't get it ?

Reply:
I gave it a new name for a section to open to the detail section as otherwise it will open up the about section

Reply:
I don't think it is a good practice to add two yields below each other, you add a yield if there is a code in between

not a good idea
```php
@yield('section1')
@yield('section2')
``` 

use it when
```php
@yield('section1')
{{-- some code in between --}}
@yield('section2')
```

Reply:
ok doc

Reply:
![maxresdefault.jpg](https://github.com/user-attachments/assets/d22fb86c-3771-4d85-9285-7e572d6b0411)